12 MolecularGlassOrientationMapping.ipynb

```diff
@@ -0,0 +1,1038 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    pss = ps.isig[:,0.25:0.35].sum(axis=-1) #Radial k-summed azimuthal intensity profiles
</pre><p>I think it would be better to define these as variables outside the function e.g.</p><p>k_min = 0.25</p><p>k_max = 0.35</p><p>then use: pss = ps.isig[:,k_min:k_max].sum(axis=-1).</p><p> </p><p>Makes it easier to change and look up the variable for other functions</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyxem/pyxem-demos/pull/84/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>